src/System.Collections/tests/Generic/Dictionary/Dictionary.Generic.Tests.ConcurrentAccessDetection.netcoreapp.cs

```diff
@@ -24,14 +24,24 @@ public class DictionaryConcurrentAccessDetectionTests
                 FieldInfo entriesType = dictionary.GetType().GetField("_entries", BindingFlags.NonPublic | BindingFlags.Instance);
                 Array entriesInstance = (Array)entriesType.GetValue(dictionary);
                 Array entryArray = (Array)Activator.CreateInstance(entriesInstance.GetType(), new object[] { ((IDictionary)dictionary).Count });
+                FieldInfo hashCode = entriesInstance.GetValue(0).GetType().GetTypeInfo().GetField("hashCode");
+                FieldInfo key = entriesInstance.GetValue(0).GetType().GetTypeInfo().GetField("key");
+                FieldInfo next = entriesInstance.GetValue(0).GetType().GetTypeInfo().GetField("next");
+                var entry = entryArray.GetValue(0);
+                hashCode.SetValue(entry, 2);
+                key.SetValue(entry, key.GetValue(entriesInstance.GetValue(0)));
+                next.SetValue(entry, 0);
+                entryArray.SetValue(entry, 0);
                 entriesType.SetValue(dictionary, entryArray);
 
+
```

Comment:
Why this extra line?

Reply:
Because entryArray is a new array not the instance from the dictionary so after initialising it to the state needed we have to write it back. The alternative is iterating and setting all the fields on the existing array entries, they're about the same amount of code.

Reply:
I meant the empty new line that was added. That it needs to be deleted to not have two empty lines. Just delete line 37 :)

Reply:
Ah! I read it in the email and missed the entire point, oops. Sorted.